SharpXMPP.Shared/XmppTcpConnection.cs

```diff
@@ -26,10 +26,10 @@ public class XmppTcpConnection : XmppConnection
         private TcpClient _client;
 
         /// <summary>
-        /// A list of name servers to use for the DNS lookup. Uses 1.1.1.1 by default. If empty, then will use the name
-        /// servers configured by the local network adapter(s).
+        /// A list of nameservers to use for the DNS lookup. If empty, then will use
+        /// the nameservers configured by the local network adapter(s).
         /// </summary>
-        public IPAddress[] NameServers { get; set; } = { IPAddress.Parse("1.1.1.1") };
+        public static IPAddress[] NameServers { get; set; } = { };
```

Comment:
I am concerned with using the static array here. Statics are generally bad.

What would you think of another design: still allowing to do quick static override, but allowing to set the servers per client of required?

```csharp
public static IPAddress[] DefaultNameservers { get; set; } = {};
public IPAddress Nameservers { get; set; } = DefaultNameservers;
```

So, we could populate from default list by default, but still allow per client override (and thus avoid any static-related problems in cases when it's important).

Reply:
We should not allow overriding nameservers in general, this case is only a workaround for Xamarin. Actually, Nameservers property should not be public, let me know if there is some modifier that exists to hide it from users but allow it to override from child package (SharpXMPP.Client.Droid)?

Reply:
Well, that kind of modifier is useless, because a user client will have different package name

Reply:
I see. Ok, good to go then.